contracts/SignatureTest.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.0;
+
+import "./ECRecovery.sol";
+import "./Eip712StructHash.sol";
+
+
+/**
+ * @title SignatureTest
+ * @dev Opens Eip712 structural hash for end-to-end tests
+ */
+contract SignatureTest {
```

Comment:
Personally I would prefer we expose more directly on the `Eip712StructHash.hash`.
If we definitely want `ECRecovery.recover`, we can expose that as well.

Reason is that if we start to do test in the same style of [open-zeppelin](https://github.com/OpenZeppelin/openzeppelin-solidity/tree/master/contracts/mocks), those can be used and naturally exposed. Also, this make this agnostic of the omg-elixir repo.

Reply:
But then I'd need to make 2 calls instead of 1 to this test-only contract. Also exposing these functions make impact on production code (RootChain contract dependencies) I'm not really feel competent. Postponing for this :pr:

Reply:
yeah, I come from a point if we are to exposing them anyway and more from a point of this repo instead of omg-elixir. But not important now and with current situation might not exactly being better as a whole.